core/controllers/contributor_dashboard.py

```diff
@@ -261,10 +262,17 @@ def get(self):
                     exp_id, language_code)
             )
         )
-
+        state_names_to_content_id_mapping_serialized = defaultdict(lambda: {})
+        for state_name in state_names_to_not_in_review_content_id_mapping:
+            content_id_to_written_translations = (
+                state_names_to_not_in_review_content_id_mapping[state_name])
+            for content_id in content_id_to_written_translations:
+                state_names_to_content_id_mapping_serialized[state_name][
+                    content_id
+                ] = content_id_to_written_translations[content_id].to_dict()
```

Comment:
Instead of reprocessing the mapping why not just serialize it from the get-go in `_get_state_names_to_not_in_review_content_id_mapping()`?